### PR TITLE
Setup CD

### PR DESCRIPTION
**Pull request by thiamsantos**
Ref #5 

- On every push on master the package will automatically be published on hex.pm.

TODO:
- [ ] Create a tag in the repo for every merge.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Contributing to vanilla-commons
```
Comment (RafaelCamarda) [inline CONTRIBUTING.md:1]:
```suggestion
# Contributing to xcribe
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+First off, thanks for taking the time to contribute!
+
+## Publish
+
+Every pull request will release a new version of xcribe on hex.pm.
+So make sure to update the changelog and bump the version before merging the PR.
+
+Run `make release` and follow the instructions. After that, update `CHANGELOG.md`
```
Comment (RafaelCamarda) [inline CONTRIBUTING.md:9]: :heart:

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+with a description of what is been changed in the release.
```
Comment (brosquinha) [inline CONTRIBUTING.md:10]: So now the release flow is to run `make release` within the dev branch; then, after merged, push the new version tag? Is that correct?

Comment (thiamsantos) [reply CONTRIBUTING.md:10]: Yes @brosquinha . A next step would be to add the `push the new version tag` to the CI flow, then everything would be automated.

**Pull request merged**